src/OSPSuite.Infrastructure.Import/Core/DataSource.cs

```diff
@@ -156,6 +156,9 @@ public bool ValidateDataSource(IReadOnlyList<ColumnInfo> columnInfos, IDimension
                      if (errorColumn.Value != null && measurementColumn.Value.Count != errorColumn.Value.Count)
                         throw new OSPSuiteException(Error.MismatchingArrayLengths);
 
+                     if (errorColumn.Value.Count == 0)
+                        throw new OSPSuiteException(Error.EmptyDataSet);
```

Comment:
don't understand the use case - how can it happen that we get 0 rows in a dataset?

Reply:
We can discuss tomorrow. It happens when all rows from a dataset are filtered out due to NaN settings. I propose we discuss what solution do we want to have for V10 and after V10.

Reply:
Can we extend the ErrorMessage with the description (=MEtadata combination) of the empty data set?
E.g. _"Your settings ended up in empty data set **'Venous Blood.Plasma.Aspirin.Day1'**. ~This could mean that for some combination of meta data values all observations are empty.~ Please remove the data set from your data or add at least one observation for it."_

Otherwise it might be hard for the user to find where the problem is.
By the way we should always try to include the description of where exactly the data is problematic. In the past our users had hard times very often, when during import of data file with very many dataset-combinations a message like "times are not unique" etc. appeared and the users tried to figure out where exactly it happened.